playbooks/robusta_playbooks/krr.py

```
@@ -296,57 +299,180 @@ def _generate_prometheus_secrets(prom_config: PrometheusConfig) -> List[KRRSecre
 
     return krr_secrets
 
+class ProcessScanParams(ActionParams):
+    scan_type: str
+    result: Any
+    scan_id: str
+    start_time: datetime
+    
+def _emit_failed_scan_event(event, scan_id, start_time, metadata, reason_msg, exception=None, result=None):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)